daml-lf/archive/src/main/protobuf/com/daml/daml_lf_dev/daml_lf_1.proto

```diff
@@ -16,21 +16,7 @@
 
 
 // Minor version history:
-// * 0 (somewhen in December 2018): initial version
```

Comment:
Not sure removing the history is usual, it seems nice to have it but I’ll leave it up to you.

Reply:
reverted.